sharedPrompts/.env.example

```
@@ -53,6 +123,40 @@ GOOGLE_GEMINI_BASE_URL=https://generativelanguage.googleapis.com
+# =========================
+# STATISTICS_CACHE_TTL=300
+
+# =========================
+# 애플리케이션 설정 (선택적 - 기본값 제공)
+# =========================
+# APP_REDIS_HOT_TTL_SECONDS=86400
+
+# =========================
+# 로깅 설정 (선택적 - 기본값 제공)
+# =========================
+# LOG_LEVEL_ROOT=INFO
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)